src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py

```diff
@@ -326,13 +372,14 @@ def content(self):
                 typ=AAZObjectType,
                 typ_kwargs={"flags": {"required": True, "client_flatten": True}}
             )
-            _builder.set_prop("properties", AAZObjectType, typ_kwargs={"flags": {"client_flatten": True}})
+            _builder.set_prop("properties", AAZObjectType, ".", typ_kwargs={"flags": {"required": True, "client_flatten": True}})

```

Comment:
Im seeing this in all aaz files which wasnt in 2022, can you check why this changed

Reply:
This was autogenerated, I will look into it